tests/TestCase/Controller/Component/AuthComponentTest.php

```diff
@@ -23,6 +23,7 @@
 use Cake\Network\Request;
 use Cake\Network\Response;
 use Cake\ORM\TableRegistry;
+use Cake\Routing\Route\InflectedRoute;
```

Comment:
Wuut? PHPStorm and many other tools don't see it that way
The separator is (should be) out of scope here regarding the order.